### DISCUSSION TITLE
Dark Mode with Inverted Mouse Pointer

### DISCUSSION THREAD

**Discussion by Onzlow** [WindowTop/WindowTop-App#361] [Q&A]
I am using WIndowTop for inverting light application windows. That is quite good working on an AMD Laptop with integrated graphics and Windows 11. 

Though I have a new laptop with Intel Iris integrated graphics also running Windows 11 23H2, just installed the latest graphics driver but when mouse overing an inverted window the inverted mouse pointer it does not work. At some points blurring or flickering but mostly not really visible it will get normal once the inverted window is left.

I have checked a lot of settings to get it working as on the first laptop. The only difference is the integrated graphics adaptor and driver from Intel vs AMD. Of course, a non-inverted mouse pointer is running fine on the Intel machine. 

Are there any ideas how to fix it? Maybe there are similar experiences?

**Comment by gileli121** [comment]
Thanks for the report.
Currently, I don't have time to try fixing the issue.

But I have an idea what you can do to resolve it.
There is an option to configure in Windows to run WindowTop with the integrated graphics.
This may solve the problem.

You can do it in Windows 11 settings.
Open Windows 11 settings and search for "graphic settings"
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/837d337f-f8a5-40d7-a00f-727ce6e71261)

If you install it not via Microsoft Store, you need to add the app manually
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/97158f07-24b0-4eda-a4f1-48dfa8e0d8c3)

If you install from Microsoft Store, you need to switch to "Microsoft Store" and search the app
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/34b0dd72-e34c-4e04-9086-3ff5db463e92)


Then, set the "Power saving" option. The power saving option will force Windows to run the app with the integrated graphics.
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/081c2530-af91-4b2c-9418-2612fd7762e1)

Once you've made the necessary changes, it's important to exit WindowTop and run it again. This step is necessary for the new settings to take effect.
Then try to use the dark mode again.

If you still have the problem, try to follow these steps, but this time, select the "High performance" option.
This is in case the problem happens with the dedicated graphic card.

Please let me know if it solves the problem.

**Reply by Onzlow** [reply]
Thank you, Gil, for the comprehensive and in-depth description of the workaround. Though I have meanwhile transferred the affected laptop to a colleague not using WindowTop.

The AMD System running WindowTop properly has the following setting:
<img width="293" alt="image" src="https://github.com/WindowTop/WindowTop-App/assets/171826728/b0cdac4b-465e-413e-ad9e-0f9531344f59">

Though while in this graphics settings menu the Windows system had used a lot of system resources.

Kind regards
Elmar
